frame/auction.ts

```diff
@@ -57,12 +61,23 @@ export async function runAdAuction(
   if (!winner) {
     return true;
   }
-  if (trustedScoringSignalsUrl !== undefined) {
-    await fetchAndValidateTrustedSignals(
-      trustedScoringSignalsUrl,
-      `keys=${[...renderingUrls].map(encodeURIComponent).join(",")}`
-    );
-  }
+  // Make all the network requests in parallel.
+  await Promise.all(
+    (function* () {
+      for (const trustedBiddingSignalsUrl of trustedBiddingSignalsUrls) {
+        yield fetchAndValidateTrustedSignals(
+          trustedBiddingSignalsUrl,
+          `hostname=${encodeURIComponent(hostname)}`
+        );
+      }
+      if (trustedScoringSignalsUrl !== undefined) {
```

Comment:
My understanding is that most interest groups are not going to bid on most requests, which means you probably do better to wait to fetch trusted scoring signals until you have run generateBid?

(Fine to go ahead with this PR as is)

Reply:
Noted; I enabled auto-merge on this PR so it's already in, but this'll change once I implement generateBid.